src/main/resources/reference.conf

```diff
@@ -25,4 +25,35 @@ spray.can {
     request-timeout = 40s
     connecting-timeout = 40s
   }
-}
\ No newline at end of file
+}
```

Comment:
Can somebody explain what the difference is between reference.conf and application.conf?